modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -71,7 +71,7 @@
     protected Set<String> reservedWords = new HashSet<String>();
     protected Set<String> languageSpecificPrimitives = new HashSet<String>();
     protected Map<String, String> importMapping = new HashMap<String, String>();
-    protected String modelPackage = "", apiPackage = "", fileSuffix;
+    protected String modelPackage = "", apiPackage = "", apiSuffix = "", fileSuffix;
```

Comment:
you forgot to define the default value of apiSuffix : 
apiSuffix = "Api"
that should resolve your CIRCLE/CI errors